### ISSUE TITLE
Reported Statistics

### ISSUE THREAD

**Issue by nklincoln** [hyperledger/caliper#912]
Currently we provide the following default transaction statistics in the reports:
- Successful transactions
- Failed transactions
- Send rate (TPS, average)
- Maximum Latency
- Minimum Latency
- Average latency
- Achieved rate (TPS, average)

It would be useful if:
- A user could name the statistics they are interested in viewing in their report
- The report could include the maximum achieved TPS

TPS trends etc (ie TPS vs time charts) could be considered, though arguably this is the job of Prometheus/Grafana

**Comment by aklenik** [comment]
Yes, this customization would be useful. As I see, the following enhancements would give us this feature easily:
1. Separate monitors and reporters: greenlisting metrics could be done in either/both components (e.g., a CSV reporter could save every metric, while an HTML reporter would only save high-level, important metrics)
2. For max/min metrics, or for time series in general, we'd need to evaluate the given metric at specific intervals during the measurement. Prometheus does exactly that, so if we provide basic counters for the TX stats, then a Prom. Monitor could give us arbitrary metrics from them. Which will be passed to the configured reporters.
